__tests__/Unit/Components/Issues/ActionForm.test.tsx

```diff
@@ -0,0 +1,64 @@
+import ActionForm from '@/components/issues/ActionForm';
+import { renderWithProviders } from '@/test-utils/renderWithProvider';
+import * as tasksApi from '@/app/services/tasksApi';
+import { fireEvent } from '@testing-library/react';
+
+describe('Issues Action Form Component', () => {
+    let updateTaskSpy: any;
+    beforeEach(() => {
+        updateTaskSpy = jest.spyOn(tasksApi, 'useUpdateTaskMutation');
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('calls updateTaskMutation on clicking assigned button', () => {
+        const screen = renderWithProviders(<ActionForm taskId="123" />);
+        const submitButton = screen.getByRole('button', {
+            name: /Assign Task/i,
+        });
+        fireEvent.click(submitButton);
+        expect(updateTaskSpy).toBeCalledTimes(2);
```

Comment:
would like to know why are we invoking `updateTaskSpy` twice.

Reply:
first call to extract mutation query, next actually calling it on button click